### DISCUSSION TITLE
Handling not supported engines

### DISCUSSION THREAD

**Discussion by jaggernoth** [macbre/sql-metadata#240] [Q&A]
Is it ok to create PR's for unsupported dialects? As long as it doesn't break existing logic of course. 
One I've got in mind is Presto's UNNEST : https://prestodb.io/docs/current/sql/select.html#unnest 
Having `unnest` in the 'FROM' section currently generates extra tables from tokens inside `UNNEST` and `t`
I can work around it in my code, but it would be much nicer to have it handled in the library itself.

**Comment by collerek** [comment]
I guess it's welcomed, that's how all backend support started😁 

If that won't break the functionality or introduce significant complication ( of already complicated) of parsing than we would be more than happy 😊
